cmd/skopeo/main.go

```diff
@@ -82,18 +82,18 @@ func main() {
 	}
 }
 
-// getPolicyContext handles the global "policy" flag.
-func getPolicyContext(c *cli.Context) (*signature.PolicyContext, error) {
-	policyPath := c.GlobalString("policy")
-	var policy *signature.Policy // This could be cached across calls, if we had an application context.
+// getTrustContext handles the global "trust" flag.
+func getTrustContext(c *cli.Context) (*signature.TrustContext, error) {
+	trustPath := c.GlobalString("trust")
+	var trust *signature.Trust // This could be cached across calls, if we had an application context.
```

Comment:
This just won’t work without corresponding changes in containers/image providing the API.